.github/workflows/publish-oci.yml

```diff
@@ -34,13 +34,18 @@ jobs:
       # This is necessary as we want to ensure that version tags
       # are properly formatted before passing them into the
       # DockerFile.
-      - name: Get CLI version
```

Comment:
I honestly have no clue how this worked in the last release given we removed the `version` script beforehand. Regardless, I moved this over to a more reliable solution 🙂